Jellyfin.Server/StartupOptions.cs

```diff
@@ -0,0 +1,41 @@
+using CommandLine;
+using Emby.Server.Implementations;
+
+namespace Jellyfin.Server
+{
+    /// <summary>
+    /// Class used by CommandLine package when parsing the command line arguments.
+    /// </summary>
+    public class StartupOptions : IStartupOptions
+    {
+        [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
+        public string DataDir { get; set; }
+
+        [Option('c', "configdir", Required = false, HelpText = "Path to use for config data (user policies and puctures).")]
```

Comment:
```suggestion
        [Option('c', "configdir", Required = false, HelpText = "Path to use for configuration data (user policies and pictures).")]
```
Btw, what's a "user policy"?

Reply:
By user policy I mean policy.xml.  The xml which records your settings like EnableRemoteAccess, EnableContentDownloading, IsAdministrator etc.  

These help text strings are only my stab at making sense of what they do.  Please suggest better strings :)

Reply:
Changed to user settings.